src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -0,0 +1,198 @@
+    LOG_BLOCK();
+
+    // NB: KV-cache is full, further generation is impossible
+    if (m_kvcache_desc.num_stored_tokens == m_kvcache_desc.total_size) {
+        OPENVINO_THROW("KV-Cache is full.");
+    }
+
+    if (m_need_copy_kvcache) {
+        LOG_VERB("Copying kv-cache from prefill to generate model.");
+        const auto kStartOutputKVCacheLayers = 1u;
+        const auto& kvcache_compiled = m_kvcache_request->get_compiled_model();
+        for (int i = 0; i < kvcache_compiled->outputs().size() - 1; ++i) {
+            const auto& output_name = kvcache_compiled->outputs()[kStartOutputKVCacheLayers + i].get_any_name();
+            auto prefill_out_tensor = m_prefill_request->get_tensor(m_prefill_out_ports.at(output_name));
+
+            const auto& input_name = std::regex_replace(output_name, std::regex("present"), "past_key_values");
+            auto kvcache_in_tensor = m_kvcache_request->get_tensor(m_kvcache_in_ports.at(input_name));
+
+            // FIXME: We don't need to fill whole tensor with 0s, but only tensor.size() - num_stored_tokens
+            //        taking into account kvcache dimension.
+            fill_tensor<ov::float16>(kvcache_in_tensor, 0);
+
+            auto prefill_out_slice =
+                make_tensor_slice(prefill_out_tensor,
+                                  m_kvcache_desc.dim,
+                                  m_kvcache_desc.max_prompt_size - m_kvcache_desc.num_stored_tokens,
+                                  m_kvcache_desc.max_prompt_size);
+
+            auto kvcache_in_slice =
+                make_tensor_slice(kvcache_in_tensor, m_kvcache_desc.dim, 0u, m_kvcache_desc.num_stored_tokens);
+
+            prefill_out_slice->copy_to(kvcache_in_slice._ptr);
+        }
+        LOG_VERB("Prepare attention mask pattern.");
+        auto* attention_mask_data =
+            m_kvcache_request->get_tensor(m_kvcache_in_ports.at("attention_mask"))->data<int64_t>();
+        attention_mask_data[m_kvcache_desc.total_size - 1] = 1;
+
+        m_need_copy_kvcache = false;
+    }
+
+    // FIXME: these tensors should be shared between the parent & child models
+    auto kv_input_ids = m_kvcache_request->get_tensor(m_kvcache_in_ports.at("input_ids"));
+    std::copy_n(input_ids->data<int64_t>(), input_ids->get_size(), kv_input_ids->data<int64_t>());
+
+    auto kv_attn_mask = m_kvcache_request->get_tensor(m_kvcache_in_ports.at("attention_mask"));
+    std::copy_n(attention_mask->data<int64_t>(), attention_mask->get_size(), kv_attn_mask->data<int64_t>());
+
+    auto kv_pos_ids = m_kvcache_request->get_tensor(m_kvcache_in_ports.at("position_ids"));
+    std::copy_n(position_ids->data<int64_t>(), position_ids->get_size(), kv_pos_ids->data<int64_t>());
+
+    m_kvcache_request->infer();
+    m_logits = m_kvcache_request->get_tensor(m_kvcache_out_ports.at("logits"));
+    m_kvcache_desc.num_stored_tokens += 1;
+
+    LOG_VERB("Write KV-cache for the new token to the correct input position for next iteration.");
```

Comment:
No VERB in runtime

Reply:
Fixed

Reply:
Sorry, it is not fixed:
![image](https://github.com/user-attachments/assets/c570a481-25fd-40d8-8582-38cc25db469d)

![image](https://github.com/user-attachments/assets/c57d5550-7dd6-4789-a632-18a7ad9e0382)


No INFO should pop in the execution logs - it will pollute the LLM execution output. No VERB too. Only debug.
So either remove all infer request completely or use DEBUG level only.

Reply:
Looks like it has been fixed